### DISCUSSION TITLE
power Tx

### DISCUSSION THREAD

**Discussion by edgaralb** [ropg/heltec_esp32_lora_v3#36] [General]
I am working with the library, but the configuration parameters generate errors (returned 0 (ERR_NONE)). Also, I need the Tx power at 10dBm, but it doesn't go above zero, so I set it to 20dBm.

mode:DIO, clock div:1
load:0x3fce3818,len:0x508
load:0x403c9700,len:0x4
load:0x403c9704,len:0xad0
load:0x403cc700,len:0x29e4
entry 0x403c9880
Radio init
[RadioLib] radio.begin() returned 0 (ERR_NONE)
Frequency: 868.10 MHz
[RadioLib] radio.setFrequency(868.1) returned 0 (ERR_NONE)
Bandwidth: 125.0 kHz
[RadioLib] radio.setBandwidth(125.0) returned 0 (ERR_NONE)
Spreading Factor: 7
[RadioLib] radio.setSpreadingFactor(7) returned 0 (ERR_NONE)
TX power: 15 dBm
[RadioLib] radio.setOutputPower(15) returned 0 (ERR_NONE)
[RadioLib] radio.startReceive(0xFFFFFF) returned 0 (ERR_NONE)

**Comment by ropg** [comment]
ERR_NONE is not an error, it is the OK, meaning "NO ERROR"
